### DISCUSSION TITLE
403: Forbidden error when registered using ORCID

### DISCUSSION THREAD

**Discussion by pibion** [CoffeaTeam/coffea-casa#316] [UNL tech support]
I registered with my ORCID ID at coffea-opendata.casa successfully, but when I try to logon I get a "403: Forbidden" error.

Is it possible ORCID identification isn't fully supported?  Two people at the workshop tried this option and both got the same error.

**Comment by oshadura** [comment]
Sorry, it is annoying feature of ORCID that emails by default are not visible by requestors. For this reason the privacy settings have to be set in ORCID so that the email attribute can be read by 'trusted' requestors.

**Reply by oshadura** [reply]
One of the users reported that you can change the primary orcid to your institutional email and made it visible directly in orcid.
